xcp/net/biosdevname.py

```diff
@@ -74,6 +74,10 @@ def all_devices_all_names():
                  "BIOS device" not in dinfo ):
                 continue
 
+            # treat USB devices as PCI
```

Comment:
Sorry.  On further consideration I'd suggest `#Treat USB devices the PCI device of their host adapter`.  There needs to be something to identify how we choose which PCI device to use.